.circleci/src/jobs/notify_release.yml

```diff
@@ -22,7 +22,8 @@ steps:
   - run:
       name: Upload backend debian packages to github release
       command: app-toolbelt v0 release upload --releaseId ${RELEASE_ID} --files "$(ls ~/attached_workspace/debs/backend/*.deb)" --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME}
-  - notify:
-      success_message: <<^ parameters.production_delivery >>[Beta] <</ parameters.production_delivery >>Ehrenamtskarte Bayern und Sozialpass Nürnberg ${NEW_VERSION_NAME} have been released successfully! https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/tag/${NEW_VERSION_NAME}-all
-      channel: releases
-      allow-all-branches: false
+# https://github.com/digitalfabrik/entitlementcard/issues/1453
```

Comment:
Do we really want to merge it like this?

Reply:
If you currently run the ci it fails. If the notify fails in other occurrences the whole pipeline fails f.e. the "delivery" and assets won't be uploaded or installed which is annoying.
Thats why i think we should spare it out until its fixed to enable delivery pipeline running successful